Project Lifecycle.md

```diff
@@ -0,0 +1,94 @@
+# Node.js Foundation Project Lifecycle
+
+## Project Definition
+
+The Node.js Foundation hosts several "Top Level Projects." These projects are autonomous from each other and governed by their own TC (Technical Committee) and chartered by the Node.js Foundation TSC.
+
+Projects are free to create "Working Groups" which are autonomous groups collaborating to fulfill a set of responsibilities. Working Groups are eventually chartered by the TC. The TSC also charters its own Working Groups.
+
+```
+ TSC
+  |
+  |-- Project A TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project TC)
+  |
+  |-- Project B TC (Chartered By TSC)
+  |       |-- Working Group (Chartered By Project TC)
+  |
+  |-- Working Group A (Chartered by TSC)
+  |-- Working Group B (Chartered by TSC)
+```
+
+Both TLPs and TSC WGs may elect a representative to the TSC. TLPs and WGs with *incubation* status are not granted voting privileges on the TSC.
+
+## Incubation
+
+The purpose of incubation is to support and mentor projects entering the foundation. The goal is for projects to be:
+
+* Participatory
+* Transparent
+* Effective
+
+While certain processes are strongly recommended because of the TSC's experience the goal of incubation is not to enforce a specific set of processes but to ensure that the processes adopted and accepted by a project achieve these goals. Therefor, the requirements for graduating from incubation are based on metrics that demonstrate success in terms of these values. These metrics are:
+
+* TC is 5 members or greater.
+* No more than 1/4 of the TC is affiliated with the same employer.
+* Members of the TC live in at least 4 different timezones.
```

Comment:
the main idea is to make sure that the project doesn't become geo-centric and adopt processes that unintentionally make it difficult for contributors in other parts of the world to be involved. for this, it's more important to specify the number of timezones as there tend to be more countries per timezone than timezones per country.